go.mod

```diff
@@ -3,16 +3,20 @@ module github.com/crossplane/crossplane
 go 1.13
 
 require (
+	cloud.google.com/go/pubsub v1.3.1 // indirect
```

Comment:
any idea why this change would be bringing in new dependencies to our build? I don't see any `*.go` files changing, so I'm not sure why we're getting new dependencies 🤔

Reply:
I can remove that. It might be after doing go mod tidy in an older version.